### PR TITLE
DFR-65 - Fix e2e tests

### PR DESCRIPTION
**Pull request by scotty2791**
Fixes # (issue)
DFR-65 - Running functional tests

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by arun-hmcts — APPROVED** [review]

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -9,4 +9,12 @@
         <gav regex="true">.*</gav>
         <cve>CVE-2018-1258</cve>
     </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+   file name: jakarta.el-3.0.3.jar
```
Comment (ruban72) [inline dependency-check-suppressions.xml:15]: Can you add a date here to suppress until a months time e.g
<suppress until="2021-09-16Z">

Comment (scotty2791) [reply dependency-check-suppressions.xml:15]: Added

Comment (ruban72) [reply dependency-check-suppressions.xml:15]: You'll need to add the date to the suppress element ie:` <suppress until="2021-09-16Z">`

Comment (scotty2791) [reply dependency-check-suppressions.xml:15]: adjusted

**Review by ruban72 — APPROVED** [review]

**Pull request merged**